sys-apps/smcipmitool/smcipmitool-2.19.0.171201.ebuild

```diff
@@ -0,0 +1,124 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eapi7-ver java-vm-2
+
+MY_DATE="$(ver_cut 4)"
+MY_PN="SMCIPMITool"
+MY_PV="$(ver_cut 1-3)"
+
+DESCRIPTION="An out-of-band utility for interfacing with SuperBlade and IPMI devices via CLI"
+HOMEPAGE="https://www.supermicro.com/"
+SRC_URI="amd64? ( ftp://ftp.supermicro.com/utility/${MY_PN}/${MY_PN}_${MY_PV}_build.${MY_DATE}_bundleJRE_Linux_x64.tar.gz )
+	x86? ( ftp://ftp.supermicro.com/utility/${MY_PN}/${MY_PN}_${MY_PV}_build.${MY_DATE}_bundleJRE_Linux.tar.gz )"
+
+LICENSE="supermicro"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE="bundled-jre bundled-stunnel"
+
+RDEPEND="sys-devel/gcc
+	sys-libs/glibc
+	sys-libs/ncurses:5
+	!bundled-jre? ( virtual/jre )
+	!bundled-stunnel? ( net-misc/stunnel )"
+
+RESTRICT="bindist fetch mirror strip"
+
+S="${WORKDIR}"
+
+QA_PREBUILT="opt/smcipmitool/libiKVM*.so opt/smcipmitool/libjcurses*.so /opt/smcipmitool/libSharedLibrary*.so opt/smcipmitool/BMCSecurity/linux/stunnel32 opt/smcipmitool/BMCSecurity/linux/stunnel64 /opt/smcipmitool/jre/*"
```

Comment:
Could you list them one per line? Just like `RDEPEND`.

Reply:
Will be fixed.